MachObfuscator/Controller/Obfuscator.swift

```diff
@@ -67,11 +53,12 @@ class Obfuscator {
             var image: Image = try! loader.load(forURL: obfuscableImage)
             image.replaceSymbols(withMap: manglingMap, paths: paths)
             // TODO: add option
+            // TODO: what is the difference between eraseSymtab and last operation in replaceSymbols?
```

Comment:
Nice catch. We probably could remove `symtab` references from `replaceSymbols` a little bit. Alternatively we could add an option for the user to decide about the obfuscation. Do you thing we should create a ticket for that, or should we just leave the TODO here?

Reply:
I implemented option for erasing (actually preserving) symtab, will push it soon.